hamster_lib/helpers/time.py

```diff
@@ -142,11 +146,12 @@ def date_time_from_groupdict(groupdict):
 
     # Individual patterns for time/date  substrings.
     relative_pattern = '(?P<relative>-\d+)'
-    time_pattern = '(?P<time>\d{2}:\d{2})'
+    time_pattern = '((?P<time>\d{2}:\d{2})|(?P<time>\d{2}:\d{2}:\d{2}))'
```

Comment:
We should probably check how regexes work:

```ipython
>>> regex.match('((?P<time>\d{2}:\d{2})|(?P<time>\d{2}:\d{2}:\d{2}))', '12:00:00').groupdict()
{'time': '12:00'}
```

Something like `(?P<time>\d{2}:\d{2}(?::\d{2})?)` will probably work better.